### DISCUSSION TITLE
Code changes not reflecting(Dev)

### DISCUSSION THREAD

**Discussion by Remi-dee** [vercel/next.js#48574] [Help]
### Summary

I’m working on an app with next js and tailwind for styling, when I try to style a component, changes aren’t reflecting on the page. I’ve tried to troubleshoot from my end, but everything seems fine with my implementation, and I’ll love to add that other changes made to my code do not reflect on the rendered page until I restart the server.

**Comment by haydenbleasel** [comment]
Hey @Remi-dee, can you share your `tailwind.config.js`? If style changes are not changing properly, it may be an issue that your `content` params are incorrect so it's not watching the right folder for changes.

**Reply by Remi-dee** [reply]
Here you go;

/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./app/**/*.{js,ts,jsx,tsx}",
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",

    // Or if using `src` directory:
    "./src/**/*.{js,ts,jsx,tsx}",
  ],
  theme: {
    extend: {
      colors: {
        amazon_blue: {
          light: "#232F3E",
          DEFAULT: "#131921",
        },
      },
    },
  },
  plugins: [require("@tailwindcss/line-clamp")],
};
